### PR TITLE
Fix widget columns configuration

### PR DESCRIPTION
**Pull request by michalskrzypek**
Fix widget columns configuration; change TextField to NumberField component

### CODE REVIEW

File: cogboard-webapp/src/components/BoardForm/index.js [hunk 1]
```diff
@@ -8,10 +8,11 @@ import { useFormData } from '../../hooks';
 import { getBoards } from '../../selectors';
 import { createValidationSchema } from './validators';
 import { trimLeadingZeros } from '../helpers';
+import NumberInput from "../widgets/dialogFields/NumberInput";
```
Comment (goeson00) [inline cogboard-webapp/src/components/BoardForm/index.js]: Add empty line between components and other imports. Components should be imported as the last ones.

File: cogboard-webapp/src/components/WidgetForm/index.js [hunk 2]
```diff
@@ -90,7 +90,7 @@ const WidgetForm = ({ renderActions, ...initialFormValues }) => {
                 shrink: true
               }}
               inputProps={{
-                min: ROWS_MIN,
+                min: WIDGET_ROWS_MIN,
                 max: 4,
```
Comment (goeson00) [inline cogboard-webapp/src/components/WidgetForm/index.js]: If we have constant for min, we should have for max if it is not dynamic.

Comment (michalskrzypek) [reply cogboard-webapp/src/components/WidgetForm/index.js]: done

File: cogboard-webapp/src/components/BoardForm/index.js (hunk 1 above)
Comment (michalskrzypek) [reply cogboard-webapp/src/components/BoardForm/index.js]: done

**Pull request merged**